portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,36 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <!DOCTYPE html>
+    <title>Chiamaka Chukwuka</title>
+    <link href="https://fonts.googleapis.com/css?family=Princess+Sofia" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Miss+Fajardose" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Ewert" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Libre+Barcode+39+Extended+Text" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Rock+Salt" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Cinzel+Decorative" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Poiret+One&display=swap" rel="stylesheet">
+    <link href="style.css" rel="stylesheet" type="text/css" />
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="logo"><em><h1 id="first"><div id="lll"><span id="c1">C</span><span id="c12">hi</span><span id="c2">&#10037;a<span id="c23">&#10037;</span></span><span id="c3">m</span><span id="c4">a</span><span id="c45">&#10037;</span><span id="c5">k</span><span id="c6">a</span></div></h1></em></div>
```

Comment:
Add some newlines+tabbing to make this more readable

Reply:
Resolving because I fixed alignment issue